tests/integration/utils2.go

```diff
@@ -1302,14 +1351,20 @@ func createDocViaGQL(
 		return nil, nil
 	}
 
-	docIDString := resultantDocs[0]["_docID"].(string)
-	docID, err := client.NewDocIDFromString(docIDString)
-	require.NoError(s.t, err)
+	docs := make([]*client.Document, len(resultantDocs))
 
-	doc, err := collection.Get(ctx, docID, false)
-	require.NoError(s.t, err)
+	for i, docMap := range resultantDocs {
+		docIDString := docMap["_docID"].(string)
```

Comment:
suggestion: Sorry about the bother, as it was a string here before, but please use `request.DocIDFieldName` instead of redefining the string value

Reply:
done